packages/next/compiled/webpack/worker.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+var _minify = _interopRequireDefault(require("./minify"));
```

Comment:
Looks like this asset was emitted without being ran through ncc. We probably need to upgrade to the new ncc with subbundle support?

Reply:
Agreed, will apply the ncc upgrade here.